docs/developing-locally-docker.rst

```diff
@@ -86,6 +86,20 @@ To migrate your app and to create a superuser, run::
 
 Here we specify the ``django`` container as the location to run our management commands.
 
+Setup your email backend
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+django-allauth sends an email to verify users (and superusers) after signup and login (if they are still not verified). To send email you need to `configure your email backend`_
```

Comment:
Towards the end of this line, I would swop, `To send email ...` with line 96, `In development ...`.

That way the idea would flow from development to production. I found that setting up e. g. Anymail with Mailgun as a default for development to be too much of a hassle. I am imagining that this is what it seems to say here since it is under *Development Mode*.